lib/dynflow/flows/atom.rb

```diff
@@ -6,7 +6,7 @@ class Atom < Abstract
       attr_reader :step_id
 
       def to_hash
```

Comment:
The naming is quite weird now... :thinking:
Should we introduce new method? e.g. `dump` or `encode`?